libbeat/outputs/elasticsearch/client.go

```diff
@@ -71,6 +71,12 @@ type Connection struct {
 	version string
 }
 
+var connectCallbackRegister connectCallback
```

Comment:
That is also getting interesting in the case we would allow in the setup to define an elasticsearch host to load the template potentially with a different user. We already have something similar for the metrics.

Talking about metrics, this means also for the connection to the monitoring instance it would load the template? Not sure if this is intended.